### ISSUE TITLE
Broken ADUnits.db

### ISSUE THREAD

**Issue by ryanhlewis** [uwescience/ADUniverse#16]
The ADUnits.db file is over the download quota, according to Github, and now the repository is broken until this file can be restored.

![brokenLFS](https://user-images.githubusercontent.com/76540311/169713734-15f56842-6b19-4617-a283-e453666205de.png)

Is there any way to host it on a Google Drive or elsewhere? I would love to work with this repository, but the burden of reverse engineering an entire database from inferring the missing columns from the code seems a bar too high.

**Comment by dhardestylewis** [comment]
@vsivsi @dhalperi @domoritz @stechu 

When you have a chance, can you see if this file can be loaded to the Seattle Open Data?
data.seattle.gov

or some other public location for download?

**Comment by dhalperi** [comment]
I don't think any of the folks you tagged have any data or power here... Perhaps reach out to UW eScience: https://escience.washington.edu/contact/

**Comment by dhardestylewis** [comment]
I appreciate y'all letting me know!

I went ahead and replied to an email from some of the PIs

**Comment by vsivsi** [comment]
Yeah, this was a Data Science for Social Good student summer project from three years ago. The students have all moved on, ~~and it looks like Nick Welch who was the Seattle City project lead is now at Microsoft~~ (incorrect, see below). So your best bet to figure out who at the City of Seattle has taken this over will be to contact the other PL, UW Professor Rick Mohler: https://arch.be.uw.edu/people/rick-mohler/

It's also possible the project DS @joseph-hellerstein may have some insights.

**Comment by dhardestylewis** [comment]
Thanks for the info! Appreciate it

**Comment by nicolaswelch** [comment]
Thanks for the explanation @vsivsi. I am still at the City and reachable at nicolas.welch@seattle.gov.

**Comment by vsivsi** [comment]
@nicolaswelch Thanks for the correction!

**Comment by lil-j** [comment]
Hello! I am curious if there was any resolved solution in being able to access the database? Thanks!
